### ISSUE TITLE
[Question]: About creating a new file

### ISSUE THREAD

**Issue by kunkeji** [infiniflow/ragflow#3762]
### Describe your problem

Do new empty files in the knowledge base need to have a file suffix? And can the newly created empty files be parsed properly?

**Comment by kunkeji** [comment]
I found that the newly created file type in the code is set to virtual file, why not create a physical file?

**Comment by KevinHuSh** [comment]
Physical file creation is well supported for OS.
Virtual file is just a name, then all the chunks in that files are added and edited manually.

**Comment by kunkeji** [comment]
but
https://github.com/user-attachments/assets/9a31b480-3c96-4556-ab8c-e3fc18e4cd4c

**Comment by kunkeji** [comment]
Okay, but currently neither the demo project nor the open source project can parse newly created blank files properly

**Comment by kunkeji** [comment]
I see, no wonder there will be an error when clicking on parse. Can't it be parsed again?
![image](https://github.com/user-attachments/assets/8b9096cf-f6a1-49e6-b8aa-4b633491e4d6)

**Comment by JinHai-CN** [comment]
Yes, but why the file can't be found? Can you reproduce this fault? @kunkeji

**Comment by kunkeji** [comment]
If it is a newly created blank document with added blocks, clicking on parse will result in an error message stating that the file cannot be found
